### PR TITLE
Edits

### PR DESCRIPTION
**Pull request by SpencerTower**
## Description

Adds redux slices for Standings and Schedule pages. Both slices query the server.
Standings page renders standings data.
Schedule page renders schedule data for games.

### Motivation and Context

Adds functionality to Standings and Schedule pages as part of Redux Toolkit refactor.

### Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: client/src/redux/slices/schedule.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    },
+});
+
+// export reducers in .actions
+export const { getInit, getScheduleSuccess } = scheduleSlice.actions;
+
+// function currying?
+export const getSchedule = (filter) => async (dispatch) => {
+
+    await wait(2000);
```
Comment (TannerSeramur) [inline client/src/redux/slices/schedule.js:40]: What's the wait for? also do you want to set loading to true before making the req?

**Comment by TannerSeramur** [comment]
Hell yeah finally someone with some motivation to contribute!

File: client/src/pages/guest/Schedule.js [hunk 2]
```diff
@@ -18,6 +17,18 @@ const Schedule = () => {
     return (
         <div>
             Show schedule here
+            {isLoading ? (
+                <p>Page Loading</p>) : (
+                <div>
+                    {games.map((game, id) => {
+                        return (
+                            <div key={id}>
+                                {`HOME TEAM: ${game.away_team} VS. AWAY TEAM: ${game.home_team}`}
```
Comment (keithj0nes) [inline client/src/pages/guest/Schedule.js:26]: The key should be the game.id. The second param of the .map is the index, and indexes should be avoided as the key - per React's documentation

File: client/src/redux/slices/schedule.js (hunk 1 above)
Comment (SpencerTower) [reply client/src/redux/slices/schedule.js:40]: Good catch, I will change it so loading is set to false before the req.

**Pull request merged**